components/google-cloud/google_cloud_pipeline_components/types/artifact_types.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 The Kubeflow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Classes for ML Metadata input/output Artifacts for tracking Google resources.
+"""
+
+from typing import Dict,  Optional
+from kfp.v2.components.types import artifact_types
+
+class VertexModel(artifact_types.Artifact):
```

Comment:
nit: suggest importing this from dsl:
```
from kfp.v2 import dsl

class VertexModel(dsl.Artifact):
  ...
```
So it's future proof just in case we move the file around.
(Our sample code only show importing these classes from dsl, I think.)

Reply:
done. thanks!